app/models/info_request.rb

```diff
@@ -804,8 +804,8 @@ def get_last_public_outgoing_event
 
     # Text from the the initial request, for use in summary display
     def initial_request_text
-        return '' if outgoing_messages.empty? # mainly for use with incomplete fixtures
-        outgoing_messages.first.get_text_for_indexing
+        body_opts = { :censor_rules => applicable_censor_rules }
```

Comment:
Better to revert to using `return '' if outgoing_messages.empty?` so that the censor rules are only loaded if its going to use them? Not sure when `applicable_censor_rules` actually gets called.